### PR TITLE
Add --osd-size parameter

### PR DESCRIPTION
**Pull request by petr-balogh**
Ticket:
https://projects.engineering.redhat.com/browse/OCSQE-258

Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

File: ocs_ci/framework/pytest_customization/ocscilib.py [hunk 1]
```diff
@@ -122,6 +122,12 @@ def pytest_addoption(parser):
             "(e.g. quay.io/rhceph-dev/ocs-olm-operator:latest-4.3)"
         )
     )
+    parser.addoption(
+        '--osd-size',
+        dest='osd_size',
+        type=int,
```
Comment (RazTamir) [inline ocs_ci/framework/pytest_customization/ocscilib.py]: 0.5TB => pass 500
2TB => pass 2000 or 2048?

Comment (petr-balogh) [reply ocs_ci/framework/pytest_customization/ocscilib.py]: documented now

**Comment by petr-balogh** [comment]
Verification running here: https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/5572/console

**Comment by petr-balogh** [comment]
I see that the device size got propagated well to the config as you can see here:
http://magna002.ceph.redhat.com/ocsci-jenkins/openshift-clusters/jnk-ai3c33-p/jnk-ai3c33-p_20200316T165042/logs/run-1584378012-config.yaml Marking as verified now as I see that value got propagated correctly.

**Pull request merged**